README.rst

```diff
@@ -265,6 +266,25 @@ NOTE: If you find problem installing ``libpng12-dev``, try installing ``libpng-d
     $ curl https://raw.githubusercontent.com/codelucas/newspaper/master/download_corpora.py | python3
 
 
+**If you are on CentOS** and use Japanese language support, install using the following:
```

Comment:
Not clear if this means:
1) Japanese language support works by default or all non-CentOS operating systems and you don't need to install
2) Japanese language support works ONLY on CentOS, and if you are on CentOS install.

If 2) is true, I think this design is not ideal. A language support should be for all operating systems, not just one because a parse/tokenizing library is available only on that OS.